sound/soc/sof/intel/hda.h

```diff
@@ -175,7 +175,7 @@
 #define HDA_DSP_STACK_DUMP_SIZE			32
 
 /* ROM  status/error values */
-#define HDA_DSP_ROM_STS_MASK			0xf
+#define HDA_DSP_ROM_STS_MASK			0xffffff
```

Comment:
GENMASK(23, 0)?
this will be an upstream comment so might as well fix it now.

Reply:
yes, changing it now.